src/figuresClasses.ts

```diff
@@ -1,11 +1,105 @@
-export interface Figure {}
+type Shape = 'triangle' | 'circle' | 'rectangle';
 
-export class Triangle implements Figure {}
+type Color = 'red' | 'green' | 'blue';
 
-export class Circle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+
+  color: Color;
+
+  a: number;
+
+  b: number;
+
+  c: number;
```

Comment:
if u want left this variant - fix indents
```suggestion
  shape: Shape = 'triangle';
  color: Color;
  a: number;
  b: number;
  c: number;
```